### PR TITLE
feature(examples) adding tracetest x aws-xray examples

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds the xray examples to work with Amazon X-Ray

## Changes

- Adds examples to the to use tracetest with xray

## Fixes

- #179 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: examples/tracetest-amazon-x-ray/src/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "author": "Tracetest",
+  "contributors": [
+    "Oscar Reyes <https://github.com/xoscar>"
+  ],
+  "dependencies": {
+    "aws-sdk": "^2.657.0",
+    "aws-xray-sdk": "^3.0.0",
+    "express": "^4.17.1",
+    "mysql": "^2.18.1"
```
Comment (jorgeepc) [inline examples/tracetest-amazon-x-ray/src/package.json:26]: `mysql`?

File: examples/tracetest-amazon-x-ray/tracetest-config.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+experimentalFeatures: []
+
+telemetry:
+  dataStores:
+    awsxray:
+      type: awsxray
+      awsxray:
+        accessKeyId: <your-accessKeyId>
+        secretAccessKey: <your-secretAccessKey>
+        region: "us-west-2"
```
Comment (jorgeepc) [inline examples/tracetest-amazon-x-ray/tracetest-config.yaml:29]: Do we need `sessionToken` here?

File: examples/tracetest-amazon-x-ray-adot/src/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "author": "Tracetest",
+  "contributors": [
+    "Oscar Reyes <https://github.com/xoscar>"
+  ],
+  "dependencies": {
+    "aws-sdk": "^2.657.0",
+    "aws-xray-sdk": "^3.0.0",
+    "express": "^4.17.1",
+    "mysql": "^2.18.1"
```
[the remark this answers is not in the document]
Comment (xoscar) [reply examples/tracetest-amazon-x-ray-adot/src/package.json:26]: LOL at some point I added something around mysql but decided to remove it 😢

File: examples/tracetest-amazon-x-ray-pokeshop/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  tracetest:
+    image: kubeshop/tracetest:${TAG:-latest}
+    platform: linux/amd64
+    volumes:
+      - type: bind
+        source: ./tracetest.config.yaml
+        target: /app/tracetest.yaml
+      - type: bind
+        source: ./tracetest.provision.yaml
+        target: /app/provisioning.yaml
```
[the remark this answers is not in the document]
Comment (xoscar) [reply examples/tracetest-amazon-x-ray-pokeshop/docker-compose.yml:12]: It seems that's the only new way to provision the tracetest server from start, I'm updating all of the examples to use this new setup

File: examples/tracetest-amazon-x-ray/src/package.json (hunk 1 above)
Comment (jorgeepc) [inline examples/tracetest-amazon-x-ray/src/package.json:26]: Should we remove this `mysql` too?

**Review by jorgeepc — APPROVED** [review]
Looks good my friend!

**Pull request merged**